extensions/default/src/hpMN.ts

```diff
@@ -0,0 +1,255 @@
+import { Types } from '@ohif/core';
+
+/**
+ * This hanging protocol can be activated on the primary mode by directly
+ * referencing it in a URL or by directly including it within a mode.
+ * It is not included in the viewer mode by default.
+ */
+const hpMN: Types.HangingProtocol.Protocol = {
```

Comment:
So I am guessing that the name MN refers to various M x N layouts - is that correct? Perhaps let's think of a more meaningful name like `hpVariousGridLayouts`? That said, if I am missing something and MN has meaning then by all means leave as is.

Reply:
How about hpMxNGrid?

Reply:
And @ohif/MNGrid - probably that means hpMNGrid would work